torchci/clickhouse_queries/compilers_benchmark_api_commit_query/query.sql

```diff
@@ -15,6 +15,11 @@ WHERE
         )
         OR empty({branches: Array(String)})
     )
+    AND NOT (
```

Comment:
Don't know if clickhouse supports functions, but if it does, can it be factored out into one?